test/render/index.js

```diff
@@ -4,7 +4,7 @@ import {SnapshotTestRunner} from '@luma.gl/test-utils';
 import EXAMPLE_TEST_CASES from './example-test-cases';
 import GLTF_TEST_CASES from './gltf-test-cases';
 
-const renderTestCaseCount = EXAMPLE_TEST_CASES.length;
+const renderTestCaseCount = EXAMPLE_TEST_CASES.length + GLTF_TEST_CASES.length;
```

Comment:
This is used to defeat tape, which has a default timeout of 500ms.

Reply:
Yes there was a helpful comment on line 10 explaining this.

I was wondering what we can do to avoid the need for such timeouts. I am assuming we'd need to provide some callback or promise to let the test framework track when testing is done?

Reply:
We do, though the timeout is a hard stop if the test framework does not hear back for a long period of time.